app/wasm/bindings/msg.go

```diff
@@ -23,3 +30,32 @@ type SwapMsg struct {
 	Route  []Step              `json:"route"`
 	Amount SwapAmountWithLimit `json:"amount"`
 }
+
+type Duration time.Duration
```

Comment:
Why is there an alias like this?

Reply:
That was my way of adding marshal/unmarshal to Duration (https://stackoverflow.com/a/48051946). Happy to change it if there's a better way.